src/EditorFeatures/Test/MetadataAsSource/MetadataAsSourceTests.cs

```diff
@@ -1653,8 +1653,6 @@ public async Task TestReadOnlyStruct()
 ' {CodeAnalysisResources.InMemoryAssembly}
 #End Region
 
-Imports System.Runtime.CompilerServices
```

Comment:
is this a desirable change?

Reply:
There seems to be a bug with the simplifier that removes imports used by attributes in the metadata as source. The AbstractImportAdder didn't add a Simplifier annotation to added imports, so this didn't cause an issue.

I don't know whether the bug should be fixed in this PR or a seperate issue created.